crates/macros/src/lib.rs

```diff
@@ -19,6 +19,9 @@ use syn::{
 struct CrossTestData {
     /// imlementations
     impls: ExprArray,
+    /// builder impl
+    #[builder(default = "syn::parse_str(\"[SimpleBuilderImplementation]\").unwrap()")]
+    builder_impls: ExprArray,
```

Comment:
I assume this will eventually be used for some marketplace builder tests?

Reply:
Yes, both here and in marketplace repos.